corehq/apps/data_interfaces/templates/data_interfaces/add_automatic_update_rule.html

```diff
@@ -5,96 +5,13 @@
 {% load crispy_forms_tags %}
```

Comment:
@gcapalbo Is this page dead code? I had to add a url for `AddAutomaticUpdateRuleView` to test it.

Reply:
it's not used anymore, it's leftover from when these had slightly different models. it's on my list to remove that page and the old models

Reply:
Cool, I'll leave removing them to you.